### PR TITLE
Add dev.atoft.Reactions

### PR DESCRIPTION
**Pull request by atoft**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: dev.atoft.Reactions.json [hunk 1]
```diff
@@ -0,0 +1,107 @@
+{
+    "app-id" : "dev.atoft.Reactions",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "41",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.ziglang"],
+    "command" : "/app/bin/reactions",
```
Comment (hfiguiere) [inline dev.atoft.Reactions.json:7]:
```suggestion
    "command" : "reactions",
```

File: dev.atoft.Reactions.json [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
```
Comment (hfiguiere) [inline dev.atoft.Reactions.json:11]:
```suggestion
        "--socket=fallback-x11",
```

File: dev.atoft.Reactions.json [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    ],
+    "modules" : [
+        {
+            "name" : "libsass",
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/lazka/libsass.git",
+                    "branch" : "meson"
```
Comment (hfiguiere) [inline dev.atoft.Reactions.json:44]: you need a commit here.

File: dev.atoft.Reactions.json [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+            ]
+        },
+        {
+            "name" : "sassc",
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/lazka/sassc.git",
+                    "branch" : "meson"
```
Comment (hfiguiere) [inline dev.atoft.Reactions.json:55]: you need a commit here.

File: dev.atoft.Reactions.json [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Dmedia-ffmpeg=enabled"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/GNOME/gtk.git",
+                    "tag" : "4.4.0",
+                    "commit" : "f1f197e3b94a55d5cbfaae2498f991a0ae733b32"
```
Comment (hfiguiere) [inline dev.atoft.Reactions.json:70]: the gtk version in the runtime isn't enough?

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build dev.atoft.Reactions

**Comment by hfiguiere** [comment]
the appdata (appstream) file is missing

File: dev.atoft.Reactions.json (hunk 5 above)
Comment (atoft) [reply dev.atoft.Reactions.json:72]: I needed to use the ffmpeg backend due to https://gitlab.gnome.org/GNOME/gtk/-/issues/4062. I don't think it's available in any runtime?

**Comment by atoft** [comment]
Thanks for the feedback :)

**Comment by atoft** [comment]
bot, build dev.atoft.Reactions

**Pull request closed without merging**